scripts/start-shared-test-validator.sh

```diff
@@ -11,6 +11,19 @@ TEST_VALIDATOR=$( cd "$(dirname "${BASH_SOURCE[0]}")/.." ; pwd -P )/.solana/acti
 TEST_VALIDATOR_LEDGER="$( cd "$(dirname "${BASH_SOURCE[0]}")/.." ; pwd -P )/test-ledger"
 FIXTURE_ACCOUNTS_DIR="$( cd "$(dirname "${BASH_SOURCE[0]}")/fixtures" ; pwd -P)"
 
+wait_for_rpc_server() {
+    while true; do
+        RESPONSE=$(curl http://127.0.0.1:8899 -s -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","id":1, "method":"getHealth"}')
```

Comment:
Fun fact, this is also available at `http://127.0.0.1:8899/health`.

Reply:
🪦 RIP